### PR TITLE
feat(lobste.rs): init

### PR DESCRIPTION
**Pull request by steinuil**
## 🎉 Theme for [Lobsters](https://lobste.rs) 🎉

Lobsters is a computing-focused community centered around link aggregation and discussion.

This is how the theme looks on my computer.

Screenshot of Lobsters on Firefox

## 💬 Additional Comments 💬

The SVG icon comes [from the lobsters repo](https://github.com/lobsters/lobsters/blob/master/app/assets/images/logo-color.svg).

Not all the CSS variables defined in the CSS file are mapped, I just changed the ones I saw were used in all the pages I tested (which is most of them).

Also, I had to use `fade()` to get the colors for the tags.

## 🗒 Checklist 🗒

- [x] I have read and followed Catppuccin's [submission guidelines](https://github.com/catppuccin/userstyles/blob/main/docs/userstyle-creation.md).
- [x] I have made a new directory underneath `/styles/<name-of-website>` containing the contents of the [`/template`](https://github.com/catppuccin/userstyles/blob/main/template/) directory.
  - [x] I have ensured that the new directory is in **lower-kebab-case**.
  - [x] I have followed the template and kept the preprocessor as [LESS](https://lesscss.org/#overview).
- [x] I have made sure to update the
      [`userstyles.yml`](https://github.com/catppuccin/userstyles/blob/main/scripts/userstyles.yml)
      file with information about the new userstyle.
- [x] I have included the following files:
  - [x] `catppuccin.user.css` - all the CSS for the userstyle, based on the
        template.
  - [x] `preview.webp` - composite image of all four individual flavor screenshots (taken with the default accent color of mauve) stitched together, generated via [Catwalk](https://github.com/catppuccin/catwalk).

### CODE REVIEW

File: styles/lobste.rs/catppuccin.user.css
Comment (uncenter) [inline styles/lobste.rs/catppuccin.user.css:135]: You shouldn't need to wrap this in the `e` function.

File: styles/lobste.rs/catppuccin.user.css
Comment (uncenter) [inline styles/lobste.rs/catppuccin.user.css:95]: Just curious what these variables theme specifically and how it looks?

**Review by uncenter — CHANGES_REQUESTED** [review]
From the preview image it looks like tags on posts aren't themed.

File: styles/lobste.rs/catppuccin.user.css
Comment (steinuil) [reply styles/lobste.rs/catppuccin.user.css:135]: Huh, when I tried it earlier, if I didn't wrap it with `e` LESS would throw an error along the lines of "invalid % without a number", but it's not doing that anymore. I'll remove it.

File: styles/lobste.rs/catppuccin.user.css
Comment (steinuil) [reply styles/lobste.rs/catppuccin.user.css:95]: Those are the post tags! This is how they look styled vs unstyled:

Screenshot 2024-11-18 at 11-21-32 Tags Lobsters

Screenshot 2024-11-18 at 11-21-55 Tags Lobsters

File: styles/lobste.rs/catppuccin.user.css
Comment (steinuil) [reply styles/lobste.rs/catppuccin.user.css]: Right, when I was testing I hadn't replaced the colors yet. Anyway, I removed it now.

**Comment by uncenter** [comment]
Would it be possible to get an invitation? I'd like to be able to check that every page is themed, even the password/login protected ones.

**Comment by steinuil** [comment]
Sure, I just need an email address. You can send me an email at steen (at) neets.work.

**Review by uncenter — CHANGES_REQUESTED** [review]
CleanShot 2024-11-19 at 17 45 57

This button is unthemed - that's it though! Once that is fixed we can merge, really nice work on this - and thanks for the invite :D

**Comment by steinuil** [comment]
All done! Thanks :)

Do the commits get squashed when the branch is merged or should I do that myself?

**Comment by uncenter** [comment]
We squash merge PRs, nothing needed from you there.

**Comment by uncenter** [comment]
We squash merge PRs, nothing needed from you there.

File: styles/lobste.rs/catppuccin.user.css
Comment (uncenter) [inline styles/lobste.rs/catppuccin.user.css:5]:
```suggestion
@version 0.1.0
```

**Review by uncenter — DISMISSED** [review]

**Comment by isabelroses** [comment]
fixed merge conflict :)

**Review by isabelroses — APPROVED** [review]

**Comment by isabelroses** [comment]
Thanks for your hard work on the new userstyle @steinuil ❤️ 

Soon after this is merged you will be invited to the userstyles maintainers team. That will grant you the ability to merge PRs for your userstyle.

Welcome to the team!

**Pull request merged**
